__tests__/app.test.js

```diff
@@ -194,6 +194,36 @@ describe("GET /api/articles", () => {
         expect(msg).toBe("bad request");
       });
   });
+  test("200: The articles can be queried to filter by topic", () => {
+    return request(app)
+      .get("/api/articles?topic=mitch")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles.length).toBe(12);
+        articles.forEach((article) => {
+          expect(article).toMatchObject({ topic: "mitch" });
+        });
+      })
+      .then(() => {
+        return request(app)
+          .get("/api/articles?topic=cats")
+          .expect(200)
+          .then(({ body: { articles } }) => {
+            expect(articles.length).toBe(1);
+            articles.forEach((article) => {
+              expect(article).toMatchObject({ topic: "cats" });
+            });
+          });
+      });
+  });
```

Comment:
Nice this is a really good start, but one extra happy path test is when given a valid topic that exists, but one with no articles what should you expect as the output?

`/api/articles?topic=paper`
(This I believe should be a topic that exists but without articles, but feel free to double check the test data if you like)